model/src/main/kotlin/AnalyzerResult.kt

```diff
@@ -46,8 +47,14 @@ data class AnalyzerResult(
         /**
          * The list of all errors.
          */
-        val errors: SortedMap<Identifier, List<Error>>
-) : CustomData() {
+        val errors: SortedMap<Identifier, List<Error>>,
+
+        /**
+         * A map that holds arbitrary data. Can be used by third-party tools to add custom data to the model.
```

Comment:
Now that we have to repeat code, should we limit yourselves to only add this to classes where we actually already make use of custom data?

Reply:
It's not so much about where we use it, but where someone might want to use it, especially now that it is possible to write plugins. If you think we should remove it from specific classes I'd propose to discuss this separately.